Formula/duc.rb

```diff
@@ -0,0 +1,39 @@
+class Duc < Formula
+  desc "Suite of tools for inspecting disk usage"
+  homepage "http://duc.zevv.nl"
+  url "https://github.com/zevv/duc/releases/download/1.4.3/duc-1.4.3.tar.gz"
+  sha256 "504810a1ac1939fb1a70bd25e492f91ea38bcd58ae0a962ce5d35559d7775e74"
+  head "https://github.com/zevv/duc.git"
+
+  option "with-opengl", "Enable OpenGL gui"
+
+  depends_on "pkg-config" => :build
+  depends_on "tokyo-cabinet"
+  depends_on "glfw" if build.with? "opengl"
+  depends_on "cairo" => :recommended
+  depends_on "pango" if build.with? "cairo"
+
+  def install
+    opts = ["--disable-x11"]
+
+    if build.with? "opengl"
+      opts << "--enable-opengl"
+      #
+    end
+    if build.without? "cairo"
+      opts << "--disable-cairo"
+      #
+    end
+
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          *opts
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/duc", "--version"
```

Comment:
Can we do something other than print version number? Like, output a graph to a file?